Qt/mainwindow.cpp

```diff
@@ -598,10 +608,13 @@ void MainWindow::createMenus()
 
 	// Help
 	MenuTree* helpMenu = new MenuTree(this, menuBar(),    QT_TR_NOOP("&Help"));
-	helpMenu->add(new MenuAction(this, SLOT(websiteAct()),    QT_TR_NOOP("Official &website"), QKeySequence::HelpContents));
-	helpMenu->add(new MenuAction(this, SLOT(forumAct()),      QT_TR_NOOP("Official &forum")));
-	helpMenu->add(new MenuAction(this, SLOT(gitAct()),        QT_TR_NOOP("&GitHub")));
-	helpMenu->add(new MenuAction(this, SLOT(aboutAct()),      QT_TR_NOOP("&About PPSSPP..."), QKeySequence::WhatsThis));
```

Comment:
Is removing the keyboard shortcut intentional?  I'm not sure anyone really needs a quick way to get to these, though...

-[Unknown]

Reply:
Yes, Windows version doesn't have them too. I can get them back.